packages/web/src/SplunkLongTaskInstrumentation.ts

```diff
@@ -52,6 +58,16 @@ export class SplunkLongTaskInstrumentation extends InstrumentationBase {
 	init(): void {}
 
 	private _createSpanFromEntry(entry: PerformanceEntry) {
+		if (
+			!!this.initOptions._experimental_longtaskNoStartSession &&
```

Comment:
This is the core of the PR... if there is no session we do not spawn new one when this flag is set to true